modules/mono/editor/bindings_generator.cpp

```diff
@@ -1403,8 +1409,13 @@ Error BindingsGenerator::_generate_cs_type(const TypeInterface &itype, const Str
 
 	if (is_derived_type && !itype.is_singleton) {
 		if (obj_types.has(itype.base_name)) {
+			TypeInterface base_type = obj_types[itype.base_name];
 			output.append(" : ");
-			output.append(obj_types[itype.base_name].proxy_name);
+			output.append(base_type.proxy_name);
```

Comment:
This changes nothing and IMO doesn't really improve readability.

Reply:
I'm just storing it in a variable so I can use it in the next line to check `is_singleton` without repeating the entire thing.

Reply:
I'm just blind and didn't notice that you actually use the variable in the very next line.